src/components/Footer.tsx

```diff
@@ -1,66 +1,47 @@
-import {
-  Box,
-  Center,
-  Container,
-  HStack,
-  Link,
-  Stack,
-  useColorModeValue,
-} from '@chakra-ui/react'
+import { Box, Container, Link, Text, VStack } from '@chakra-ui/react'
 import Image from 'next/image'
 
 import poweredByVercelLogo from '@app-public/powered-by-vercel.svg'
-import { REPO_URL } from '~/common'
 
 const Footer = () => {
-  const boxBgColor = useColorModeValue('gray.50', 'gray.900')
-
   return (
-    <Box as="footer" bg={boxBgColor} flexShrink={0}>
-      <Container py={5} variant="fluid">
-        <Stack
-          justify="space-between"
-          direction={['column', 'row']}
-          spacing={4}
-        >
-          <HStack
-            shouldWrapChildren
-            justify="space-between"
-            alignItems="center"
-            justifyContent="center"
-            fontSize={{ base: 'md', md: 'lg' }}
+    <Box as="footer" bgColor="background2">
```

Comment:
The footer should have the same color as the page background.

It looks fine on the homepage, but it looks bad on the comparator page.

Reply:
I see, I'll have to tackle that. On the other hand, by having the very same color from the page, it's difficult to identify the footer visually. Shouldn't have a slightly different tone to make it easier to be identified?

Reply:
My assumption is that people will know it's the end of the page, and that they've reached the footer because the text and Vercel logo are centered instead of left-aligned, and because you cannot scroll any further down the page.

My idea for the footer is that it should not be grabbing too much attention. It's there if you're interested, but otherwise it should just "lay flat" on the page.

The links with our names have a nice dash of color, and the Vercel badge (for me) clearly denotes the end of the page.

But if you really want, I can live with a slight background difference as well.

Reply:
Fair enough, let's keep it that way. I'll fix the bg color for the comparator page later!